app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -44,11 +51,13 @@ class Uploader extends \Magento\Backend\Block\Widget
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Framework\File\Size $fileSize,
+        \Magento\Framework\Image\Adapter\ConfigInterface $imageConfig,
```

Comment:
Hi, @eduard13, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add new required parameters to the constructor. 

Please make this param optional, the example you can look [here](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)